pennotools/contributie/process.py

```diff
@@ -88,8 +92,25 @@ def write_contributie(workbook: Workbook,
                'endtoendid'
                ]
 
-
-def get_sepa_rows(p: Person, value: Decimal, description, split=Decimal('100.00')) -> List[Dict]:
+# Refer to https://www.rabobank.nl/images/pdf_formaatbeschrijving_csv_29856100.pdf
```

Comment:
Is het alleen voor SEPA Direct Debit imports? Dan kan je misschien beter de standaard XML SEPA format gebruiken (pain.008.001.02): https://www.rabobank.com/nl/images/formaatbeschrijving-xml-sepa-direct-debit-okt.pdf, dan is het niet Rabobank specifiek. Er is evt ook een library voor (maar dat is hier vast niet zo handig): https://github.com/raphaelm/python-sepaxml

Reply:
Ja, de XML zou beter zijn denk ik maar ik ben iets te lui om dat helemaal uit te zoeken haha. We betaalde vroega ook aan een website dat die specificatie helemaal bijhoud en update per bank zodat het correct was voor de ABN en we een mooi XML SEPA kregen van onze output.
Dit is ook het formulier dat aan de Rabobank gegeven moet worden volgens Erwin.

Reply:
Op zich kan de Penno ook zo makkelijker checken of alles klopt. Die XML is een beetje onduidelijk